src/state/index.tsx

```diff
@@ -18,6 +18,12 @@ export interface StateContextType {
   setError(error: TwilioError | null): void;
   notification: string | null;
   setNotification(notification: string | null): void;
+  isWaitingForRoomToStart: boolean;
```

Comment:
You don't use these anywhere so please remove isWaitingForRoomToStart and setIsWaitingForRoomToStart